agentMET4FOF/agents.py

```diff
@@ -456,18 +456,30 @@ def send_plot(self, fig=plt.Figure()):
         fig : Figure
             Can be either matplotlib figure or plotly figure
 
+        mode : str
+            "image" - converts into image via encoding at base64 string.
+            "plotly" - converts into plotly figure using `mpl_to_plotly`
+            Default: "image"
+
         Returns
         -------
         The message format is {'from':agent_name, 'plot': data, 'senderType': agent_class}.
         """
+
         if isinstance(fig, matplotlib.figure.Figure):
-            #graph = self._convert_to_plotly(fig) #unreliable
-            graph = self._fig_to_uri(fig)
+            if mode == "plotly":
+                graph = self._convert_to_plotly(fig)
+            elif mode == "image":
```

Comment:
If we did the input check above, we should switch this to else and insert a short inline comment about the value being "image". Otherwise we should insert an else statement and throw an exception or at least a warning.